cScripts/Loadouts/CfgLoadouts_Charlie.hpp

```diff
@@ -117,7 +120,7 @@ class CAV_Charlie_JFO : CAV_Charlie_Base {  // Joint Fires Observer 36.3kg
 class CAV_Charlie_SL : CAV_Charlie_Base {  // Squad Leader
 
     headgear[] = {"rhsusf_ach_helmet_headset_ess_ocp"};
-    backpack[] = {"B_Carryall_mcamo"};
+    backpack[] = {"B_AssaultPack_mcamo"};
```

Comment:
Can be removed due to being defined in Charlie base as of #267

Reply:
updated